lib/traject/macros/iiif.rb

```diff
@@ -0,0 +1,80 @@
+# frozen_string_literal: true
+
+module Macros
+  # Macros for extracting Stanford Specific MODS values from Nokogiri documents
+  module Stanford
+    def grab_sul_iiif_links(manifest)
+      manifest = Net::HTTP.get(URI.parse(manifest))
+      JSON.parse(manifest)
+    rescue
+      nil
+    end
+
+    def process_iiif_thumbnail(manifest_json)
+      manifest_json['thumbnail']['@id']
+    rescue JSON::ParserError
```

Comment:
Seeing as the json is already passed in, is there some reason to expect a ParserError here? (same question applies to several methods below)